regretBlocker.js

```diff
@@ -0,0 +1,9 @@
+// @flow
```

Comment:
Good thinking! And yes, without the comment [Flow](https://flowtype.org/) will not process the file.